webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/LdapQueryProcessor.scala

```diff
@@ -217,6 +217,19 @@ class AcceptedNodesLDAPQueryProcessor(
     } }
   }
 
+  override def processOnlyId(query:Query) : Box[Seq[NodeId]] = {
+    //only keep the one of the form Full(...)
+    queryAndChekNodeId(query, NodeInfoService.nodeInfoAttributes, None).map { seq => seq.flatMap {
+      case QueryResult(nodeEntry, _ , _) =>
```

Comment:
I'm not sure you need to get all these attributes here, they don't seem to be use. 
I believe you can juste request `NODE_UUID`